_pages/API/v1.8/DataService/026-dataservice-measurementsandvalues.md

```diff
@@ -380,6 +382,7 @@ You can fetch all measurements with values or only certain measurements with val
 <nobr><code>DateTime</code> fromModificationDate </nobr> | Specifies a date to select all measurements that where modified after that date. Please note that the system modification date (lastModified property) is used and not the time attribute (creation date).
 <nobr><code>DateTime</code> toModificationDate </nobr> | Specifies a date to select all measurements that where modified before that date. Please note that the system modification date (lastModified property) is used and not the time attribute (creation date).
 <nobr><code>int</code> limitResult </nobr>| Restricts the number of result items. <br> `limitResult=100`
+<nobr><code>int</code> limitResultPerPart </nobr>| Restricts the number of result items per part. This parameter only takes effect in combination with either partPath and deep or partUuids:<br> PartPath & deep: trigger a deep search returning at max limitResultPerPart measurements for each child part.<br> PartUuids: return at max limitResultPerPart measurements for each specified part.<br> `limitResultPerPart=20`
```

Comment:
I recommend to write "PartPath & Deep" to indicate those two belong together. Otherwise it is a bit harder to understand that "and" has a highter priority than "or".

Reply:
Done